src/core/src/bound_evaluate.cpp

```diff
@@ -474,7 +474,7 @@ bool ov::tensor_is_positive(const Tensor& bound) {
     const auto zero_constant = op::v0::Constant::create(bound.get_element_type(), {1}, {0});
     OutputVector greater(1);
 
-    bool folded = std::make_shared<op::v1::Greater>(bound_constant, zero_constant)
+    bool folded = std::make_shared<op::v1::GreaterEqual>(bound_constant, zero_constant)
```

Comment:
method is called as `tensor_is_possitive`. Now it gets `tensor_is_non_negative`.
Also, can we have unit-tests for ReduceProd `evaluate` to cover this case?